STM32F1/libraries/A_STM32_Examples/examples/Sensors/HardTimerAsEncoder/HardTimerAsEncoder.ino

```diff
@@ -26,92 +26,54 @@
  * mode 2 - only counts pulses on Channel A
  * mode 3 - counts on both channels. 
  * 
-*/
-
-
+ */
 
 #include "HardwareTimer.h"
+#define SOUT Serial
```

Comment:
If you want to use Serial3 instead of normal Serial, you can do that switch here in one single line, instead of having to find-and-replace the entire file. Come to think about it... maybe that's what Serial already is, - a macro for Serial1. I'll have to investigate that

Reply:
Yes, Serial can be re-defined in an .ino accomplishing exactly the same thing. I'll remove SOUT.